cmd/node/node.go

```diff
@@ -871,7 +871,7 @@ func (app *SpacemeshApp) Start(cmd *cobra.Command, args []string) {
 	lg := log.NewDefault(nodeID.ShortString())
 
 	apiConf := &app.Config.API
-	dbStorepath := app.Config.DataDir
+	dbStorepath := app.Config.DataDir()
```

Comment:
I figured you could shrink this whole pr by adding a line here that appends the netid to the storepath.. you would've probably need to add it in a few more places to support the other binaries though.

Reply:
I also do the canonical path thing (turning `~` into the user's home dir) and I found this easier to do.